### ISSUE TITLE
Tags on EventHubNamespace do not work properly

### ISSUE THREAD

**Issue by chdalski** [terraform-providers/terraform-provider-azurerm#12003]
Hi there,

azurerm doesn't fetch the tags from an eventhub_namespace as expected.

Please see below for details.

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version

Verifying Terraform installation...
/opt/hostedtoolcache/terraform/0.15.4/x64/terraform version
Terraform v0.15.4
on linux_amd64

Initializing provider plugins...
- Finding latest version of hashicorp/azurerm...
- Installing hashicorp/azurerm v2.61.0...
- Installed hashicorp/azurerm v2.61.0 (signed by HashiCorp)

### Affected Resource(s)

* azurerm_eventhub_namespace

### Terraform Configuration Files

```
resource "azurerm_eventhub_namespace" "event_hub_namespace" {
  name                     = "example"
  resource_group_name      = var.resource_group_name
  location                 = var.location
  sku                      = "Standard"
  capacity                 = 1
  auto_inflate_enabled     = true
  maximum_throughput_units = 4

  tags = {
     Environment       = "test"
     ResourceGroupName = "TEST-RG"
  }
}
```
### Expected Behaviour

"terraform apply" applies the tags correctly

after applying the next "terraform plan" should show no changes

### Actual Behaviour

"terraform apply" applies the tags correctly

after applying the next "terraform plan shows:
```
  # azurerm_eventhub_namespace.event_hub_namespace will be updated in-place
  ~ resource "azurerm_eventhub_namespace" "example" {
        id                                  = "/subscriptions/XYZ/resourceGroups/TEST-RG/providers/Microsoft.EventHub/namespaces/example"
        name                                = "example"
      ~ tags                                = {
          ~ "Environment"       = "TEST-RG" -> "test"
            # (1 unchanged element hidden)
        }
        # (12 unchanged attributes hidden)
    }
```
Somehow the tags are incorrectly read by azurerm from the eventhub_namespace.
The tags are correctly displayed in Azure Portal.

**Comment by alex-3sr** [comment]
Hi,

I can confirm the whole point. it detect now drift with all tags since we updated last version 2.61

Thanks
Alexandre

**Comment by ces16** [comment]
The problem isn't present in version 2.60.

**Comment by drdamour** [comment]
i thought maybe this was our blueprint doing something bad..but guess it's terraform.  No plan comes back as clean now :(

**Comment by tombuildsstuff** [comment]
Fixed via #12062
